metron-platform/metron-data-management/pom.xml

```diff
@@ -178,32 +185,9 @@
         </dependency>
         <dependency>
             <groupId>org.apache.hbase</groupId>
-            <artifactId>hbase-common</artifactId>
-            <version>${global_hbase_version}</version>
-            <exclusions>
-              <exclusion>
-                <groupId>org.apache.hadoop</groupId>
-                <artifactId>hadoop-common</artifactId>
-              </exclusion>
-            </exclusions>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.hbase</groupId>
-            <artifactId>hbase-server</artifactId>
+            <artifactId>hbase-mapreduce</artifactId>
```

Comment:
In the newer version of HBase, we can pull in `hbase-mapreduce` rather than all of `hbase-server`.  The `hbase-mapreduce` package did not exist in older versions of Hbase.